### ISSUE TITLE
Only build no install

### ISSUE THREAD

**Issue by jonas-w** [Morganamilo/paru#970]
I checked the issues and documentation and couldn't find anything about it, sorry if this is a duplicate.

Would it be possible to only build the packages into the localrepo but not install it? My use case would be to pre-build some packages on my server and install them on another machine using the repo that paru created.

Basically a flag/config option, that skips the part where paru installs the package.

**Comment by Morganamilo** [comment]
--noinstall when combined with --chroot will do what you want.

**Comment by Morganamilo** [comment]
Well without --chroot paru will still need to install dependencies. Though if you're running this in docker then "your system" in this context is already a container so I guess it doesn't matter.

**Comment by kamack38** [comment]
@Morganamilo There's no such flag as `--noinstall`.
```
$ paru -S paru --noinstall
error: unknown option --noinstall
```

**Comment by parkerlreed** [comment]
--noinstall isn't included in 1.11.1 (committed 3 days afterwards) so if your paru isn't already up to date it won't be available.

(I only noticed this since Steam OS 3.0 ships 1.11.1)

**Comment by kamack38** [comment]
I still don't understand how to build a package to a selected directory. Could you please provide an example?
